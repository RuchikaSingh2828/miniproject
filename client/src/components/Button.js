import React from 'react';
import { Button } from 'react-bootstrap';

 const button = ({ text, handler }) => {
    return <Button className="button centered" onClick={ handler }>{ text }</Button> 
}

export default button;