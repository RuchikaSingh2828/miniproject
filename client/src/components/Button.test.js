import React from 'react';
import renderer from 'react-test-renderer';
import Button from './Button';

test('Button rendering test', () => {
    const btn = renderer.create(
        <Button text="Click Me" handler={()=>{}} />
    );

    let tree = btn.toJSON();
    expect(tree).toMatchSnapshot();
})