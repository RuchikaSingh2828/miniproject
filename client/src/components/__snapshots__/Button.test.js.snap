// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button rendering test 1`] = `
<button
  className="button centered btn btn-primary"
  disabled={false}
  onClick={[Function]}
  type="button"
>
  Click Me
</button>
`;
